browserstack/scots_lidar_bs_tests.py

```diff
@@ -0,0 +1,162 @@
+from decouple import config
+from dotenv import load_dotenv
+import re
+import time
+from capabilities import browsers
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options as ChromeOptions
+from selenium.webdriver.firefox.options import Options as FirefoxOptions
+from selenium.webdriver.safari.options import Options as SafariOptions
+from selenium.webdriver.edge.options import Options as EdgeOptions
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.common.by import By
+from selenium.common.exceptions import NoSuchElementException
+from threading import Thread
+
+load_dotenv()
+
+BROWSERSTACK_USERNAME = config('BROWSERSTACK_USERNAME')
+BROWSERSTACK_ACCESS_KEY = config('BROWSERSTACK_ACCESS_KEY')
+BROWSERSTACK_URL = config('BROWSERSTACK_URL')
+
+BUILD_NAME = "Scots LiDAR {} Testing"
+URL_UNDER_TEST = "https://remotesensingdata.gov.scot/"
```

Comment:
I think it would be worth setting this as a .env variable at some point. If the next step is to run the tests locally or on Jenkins before deploy, then we'll need a configurable URL. An "ENV" variable (e.g. "dev", "beta", "live") which becomes part of the build name might also be useful so you can see on the browserstack site what kind of tests are being run.

Reply:
done!